pyanaconda/modules/security/installation.py

```diff
@@ -399,23 +399,27 @@ def name(self):
         return "Configure fingerprint authentication"
 
     def _is_fingerprint_configuration_supported(self):
+        """Is the fingerprint configuration supported?"""
         return (os.path.exists(self._sysroot + PAM_SO_64_PATH) or
                 os.path.exists(self._sysroot + PAM_SO_PATH))
 
     def run(self):
+        """Run the task."""
         if not self._fingerprint_auth_enabled:
+            log.debug("Fingerprint configuration is not enabled. Skipping.")
             return
 
         if not self._is_fingerprint_configuration_supported():
             log.debug("Fingerprint conifguration is not supported on target system.")
```

Comment:
Maybe this line could also say Skipping, same as the case above?

Reply:
Fixed.